data/test_survey_structure.xml

```diff
@@ -0,0 +1,832 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This XML Structure is given by LimeSurvey by exporting surveys right?

Reply:
Genau.